setup.py

```diff
@@ -65,6 +65,7 @@
         "eth-ape>=0.3.0,<0.4.0",
         "ethpm-types",  # Use the version ape requires
         "packaging>=20.9,<21",
+        "requests",
```

Comment:
Do we use `requests` or is it implicitly installed from one of the other dependencies?

Reply:
we import an exception from it and Pycharm was complaining that it was not listed so I added it. Otherwise it was always implicitly installed. It does not hurt to include, since technically we are using it in our sources files this way.